internal/staticupload/get.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package staticupload
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/edgelesssys/constellation/v2/internal/kms/storage"
+)
+
+// Get is a convenience method to return a DEK from from AWS S3 Storage by key ID.
+func (s *Client) Get(ctx context.Context, keyID string) ([]byte, error) {
+	getObjectInput := &s3.GetObjectInput{
+		Bucket: &s.bucketID,
+		Key:    &keyID,
+	}
+	output, err := s.s3Client.GetObject(ctx, getObjectInput)
+	if err != nil {
+		var nsk *types.NoSuchKey
+		if errors.As(err, &nsk) {
+			return nil, storage.ErrDEKUnset
+		}
+		return nil, fmt.Errorf("downloading DEK from storage: %w", err)
+	}
+	return io.ReadAll(output.Body)
+}
```

Comment:
This convenience method is very specific to one part of our code base. I would not define it here and instead make it a function in the package where it is used.

Reply:
ok